### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by wilsonianb**
+r

### CODE REVIEW

File: ansible/roles/validation-tracker/templates/upstart.conf.j2 [hunk 1]
```diff
@@ -9,8 +9,9 @@ env HBASE_HOST={{ hbase_host }}
 env HBASE_PORT={{ hbase_port }}
 env HBASE_TABLE={{ hbase_table }}
 env VALIDATION_TRACKER_POSTGRES_URL={{ postgres_url }}
+HTTP_SERVICE_URL={{ http_service_url }}
 
-exec validation-tracker --hbase --postgres
+exec validation-tracker --hbase --postgres --http
```
Comment (wilsonianb) [inline ansible/roles/validation-tracker/templates/upstart.conf.j2]: Do we want to continue writing to hbase and old postgres?

**Comment by stevenzeiler** [comment]
Is the `http_service_url` ansible variable documented?

File: ansible/roles/validation-tracker/templates/upstart.conf.j2 (hunk 1 above)
Comment (stevenzeiler) [inline ansible/roles/validation-tracker/templates/upstart.conf.j2]: no

**Comment by wilsonianb** [comment]
No. I added it to the development.yml vault file. What is a good place to document all of these ansible variables?

**Comment by stevenzeiler** [comment]
How about in the default playbook? https://github.com/ripple/validation-tracker/blob/master/ansible/playbook.yml

**Comment by wilsonianb** [comment]
DOH

**Comment by wilsonianb** [comment]
Variable documented

**Comment by stevenzeiler** [comment]
:+1: LGTM

**Pull request merged**